src/shared/prerendered-app/Intro/style.css

```
@@ -118,6 +118,93 @@
   fill: var(--light-blue);
 }
 
+.bottom-wave {
+  position: relative;
+}
+
+.info {
+  background: var(--light-gray);
+  position: relative;
+  padding: 5em 2em;
+  @media (min-width: 1200px) {
+    padding: 5em 0;
+  }
+}
+
+.info-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  opacity: 0;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 40px);
+    transition: opacity, transform 300ms ease-in;
+  }
+}
+
+.info-container.appear {
+  opacity: 1;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 0px);
+  }
+}
+
+.info-content {
+  display: grid;
+  align-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
```

```suggestion
  grid-template-columns: repeat(auto-fit, minmax(min(100%, 320px), 1fr));
```
fixes small viewport horizontal overflow